src/parser/GenericParser.test.ts

```diff
@@ -191,6 +191,16 @@ describe("GenericParser", () => {
             );
         });
 
+        it("should count correctly for an empty file", async () => {
+            const inputPath = fs.realpathSync(phpTestResourcesPath + "empty.php");
+            const parser = new GenericParser(getParserConfiguration(inputPath));
+            const results = await parser.calculateMetrics();
+
+            expect(results.fileMetrics.get(inputPath)?.get("real_lines_of_code")?.metricValue).toBe(
+                0
+            );
```

Comment:
I just thought (because we have many such tests) that this could be shortened like this (pseudo code):
```suggestion
            const { fileMetrics } = await calculateMetrics(phpTestResourcesPath + "empty.php");

            expect(fileMetrics.get(inputPath)?.get("real_lines_of_code")?.metricValue).toBe(
                0
            );
```

Note: `calculateMetrics` would be a new method that creates the GenericParser and calls `realpathSync`

Reply:
@mylinhdao this could be interesting to you, I guess :-)

Reply:
Something similar to is already done by `testFileMetrics(...)` in #42. These new test cases will have to be refactored to also use `testFileMetrics` once merged. And put into the right file, too. Merge will be wonderful :D